### DISCUSSION TITLE
Meetings table width cuts off "Type" column

### DISCUSSION THREAD

**Discussion by Spsrky** [code4recovery/12-step-meeting-list#696] [Q&A] [answered]
Recently switched from SiteOrigin page builder to plain vanilla Gutenberg and now the Meetings table is too narrow to display the "Meeting Type" column. I tried changing  width for various DIV elements in the page using the inspector but either can't find the right one or that method just doesn't work. Changing "Page Layout" in the Customizer from "No Sidebar" to "Full Width" causes filter buttons to wrap but doesn't affect the table width. Of course, there's no way to get into the block editor to change the parameters for whatever block the table is in. Thanks for any help!

Theron B

**Comment by brianw-area24** [comment]
Are you using the Gutenberg plugin, or just the editor that's included with Wordpress by default? If it's the Gutenburg plugin, you might try deactivating that plugin. A while back we had a few sites that needed to deactivate the Gutenberg plugin in order to display properly.

If you continue having issues, it might help us if you can include the URL of your site here.

Brian W.

**Comment by Spsrky** [comment]
Hi Brian,

Thanks for the quick response! We're using the built-in block editor plus a couple of plugins that provide additional blocks. I tried deactivating the plugins but that made no difference in how the Meetings table is displayed. The Meetings page URL is https://www.pdxaa.org/index.php/meetings/. The inspector shows that DIV#container has left and right margins of 286px each so I was trying to free up some of that space but wasn't able. Maybe edit page template code?

In fellowship and service,

Theron

**Comment by Spsrky** [comment]
A little more info, FWIW - I tried previewing a couple of the stock WorkPress themes in Customizer and they worked fine - all table columns were visible. Then I switched back to our active theme, SiteOrigin North, and in the Customizer it worked fine, too. Hmmm...

In fellowship and service,

Theron


On Feb 18, 2022, 8:14 PM -0800, Portland Intergroup Webmaster ***@***.***>, wrote:
	Hi Brian,

Thanks for the quick response! We're using the built-in block editor plus a couple of plugins that provide additional blocks. I tried deactivating the plugins but that made no difference in how the Meetings table is displayed. The Meetings page URL is https://www.pdxaa.org/index.php/meetings/. The inspector shows that DIV#container has left and right margins of 286px each so I was trying to free up some of that space but wasn't able. Maybe edit page template code?

In fellowship and service,

Theron


On Feb 18, 2022, 6:19 PM -0800, brianw-area24 ***@***.***>, wrote:
	Are you using the Gutenberg plugin, or just the editor that's included with Wordpress by default? If it's the Gutenburg plugin, you might try deactivating that plugin. A while back we had a few sites that needed to deactivate the Gutenberg plugin in order to display properly.
If you continue having issues, it might help us if you can include the URL of your site here.
Brian W.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Spsrky** [comment]
Correction - in Customizer, the "Any Type", "List" and "Map" buttons show at the top of the list but no "Type" column, regardless of which theme used. Thought I might have been onto something - oh well.

Theron

**Reply by kiyote33** [reply]
I don't recall the Type column showing by default. Our WIKI Frequently Asked Questions provides the following advice: 
To see types in the meeting list, one way to do it is to add some CSS to your theme which will make a types column visible.
**	@media screen and (min-width: 768px) {
		div#tsml #meetings .types { display: table-cell !important; }
	}**
So, if you add **@media screen and (min-width: 768px) {div#tsml #meetings .types { display: table-cell !important; }}** to the Additional CSS in the Customizer, you should see your Type column back to the right of Region...

**Comment by Spsrky** [comment, marked as answer]
Thanks, Wayne! To tell the truth, I don't recall either whether that column was showing before I made the change to the block editor, but our Intergroup office manager told me it was missing. I've added the CSS you suggested and that did the trick, so all is good. Thanks!

Theron
